pkg/swagger/crd.go

```diff
@@ -0,0 +1,138 @@
+package swagger
+
+import (
+	"bytes"
+	_ "embed"
+	"io"
+	"strings"
+
+	goyaml "gopkg.in/yaml.v3"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	"k8s.io/apimachinery/pkg/util/yaml"
+)
+
+//go:embed objectmeta.json
+var objectmeta []byte
+
+type CRDLoader struct {
+	objectMetaDefinitions *Schema
+}
+
+func (c *CRDLoader) Load(manifest []byte) (Definitions, error) {
+	crds, err := c.splitYAML(manifest)
+	if err != nil {
+		return nil, err
+	}
+
+	if err := c.loadObjectMeta(); err != nil {
+		return nil, err
+	}
+
+	definitions := []apiextensionsv1.CustomResourceDefinition{}
+	for _, crdfile := range crds {
+		crd := apiextensionsv1.CustomResourceDefinition{}
+
+		decoder := yaml.NewYAMLOrJSONDecoder(bytes.NewBuffer(crdfile), 4096)
+		if err := decoder.Decode(&crd); err != nil {
+			panic(err)
```

Comment:
panic? Returning an error should do right?
```suggestion
			return nil, err
```

Reply:
Haha, a leftover from my private POC, nice catch.